contrib/automation_tests/test_cases/top_down_tab.py

```diff
@@ -5,58 +5,52 @@
 """
 import logging
 
+from core.common_controls import Table
 from core.orbit_e2e import E2ETestCase, find_control
 
 
 class VerifyHelloGgpTopDownContents(E2ETestCase):
```

Comment:
Same as for the Bottom Up test, please add a comment

Reply:
Done.